### ISSUE TITLE
load fails on Windows machines if user has a space in their user name

### ISSUE THREAD

**Issue by ellisp** [christophsax/seasonal#146]
I know I know, I shouldn't have a space in my user name and I never will again... but it's bound to ban issue for people.  See below.  There's probably a fix in including the command in quotes or something.
```
> library(seasonal)
Error : .onLoad failed in loadNamespace() for 'seasonal', details:
  call: x13binary::checkX13binary(verbose = FALSE)
  error: When running

  C:/Users/Peter Ellis/Documents/R/win-library/3.2/x13binary/bin/x13ashtml.exe

Command Prompt returned the following message:

  'C:/Users/Peter' is not recognized as an internal or external command,
  operable program or batch file.
```

**Comment by christophsax** [comment]
My bad. This existed in seasonal before, was corrected, but re-introduced again in x13binary. Fixed it there. Can you try:

```
devtools::install_github("x13org/x13binary", ref = "bug/quote-windows")
x13binary::checkX13binary()
```
If this works, seasonal should work as well.

**Comment by jbkunst** [comment]
That missing quotes :smile: 

I tried but have the 127 issue.

```
Downloading GitHub repo x13org/x13binary@bug/quote-windows
from URL https://api.github.com/repos/x13org/x13binary/zipball/bug/quote-windows
Installing x13binary
"C:/PROGRA~1/R/R-32~1.3/bin/x64/R" --no-site-file --no-environ --no-save --no-restore CMD INSTALL "C:/Users/Joshua  \
  K/AppData/Local/Temp/RtmpiQy0rj/devtools7e445111ceb/x13org-x13binary-56b46ca" --library="C:/Program Files/R/R-3.2.3/library" --install-tests 

* installing *source* package 'x13binary' ...
Warning: running command 'sh ./configure.win' had status 127
ERROR: configuration failed for package 'x13binary'
* removing 'C:/Program Files/R/R-3.2.3/library/x13binary'
* restoring previous 'C:/Program Files/R/R-3.2.3/library/x13binary'
Error: Command failed (1)
```

Anyway I fork the `x13binary`  repo, add the quote and work ;)!

**Comment by christophsax** [comment]
Hi Josh, thanks. I looked at the changes in your fork, but they seem to be more or less the same what I [did](https://github.com/x13org/x13binary/blob/bug/quote-windows/R/checkX13binary.R). (Except the `normalizePath` and changing the working directory, which should solve another issue).

The error you saw has encountered during installation and before running `checkX13binary`. Could it be something with the devtools installation? If you have time, could you try to clone install the [quote](https://github.com/x13org/x13binary/tree/bug/quote-windows) branch locally at some point? Would be great if this could be resolved.

**Comment by brookstagg** [comment]
Spaces are getting me down. My challenge is in the space between `Program` and `Files`. I seen elsewhere a workaround is to install R and Rstudio to root C:, but was hoping to avoid that. Thanks
```C:/Program Files/R/R-3.2.3/library/x13binary/bin/x13ashtml.exe

Command Prompt returned the following message:

  'C:/Program' is not recognized as an internal or external command,
  operable program or batch file.```

**Comment by christophsax** [comment]
Have you tried this one?

```
devtools::install_github("x13org/x13binary", ref = "bug/quote-windows")
x13binary::checkX13binary()
```

Let me know if it works.

**Comment by brookstagg** [comment]
Sorry, i should have mentioned it. I did, and it did not work. The new install worked but 
    x13binary::checkX13binary(verbose = FALSE)

did not.

**Comment by christophsax** [comment]
Can you poste me the output:

```
 x13binary::checkX13binary(verbose = TRUE)
```

**Comment by christophsax** [comment]
And you really need the latest version: So you may try again:

```
devtools::install_github("x13org/x13binary")
```

**Comment by brookstagg** [comment]
Ugh. I'm so sorry, I didn't read the output close enough. i figured it out and will leave this here as a note. 

Innitially the reinstall of `x13binary` failed. (see below). 

    Downloading GitHub repo x13org/x13binary@bug/quote-windows
    from URL https://api.github.com/repos/x13org/x13binary/zipball/bug/quote-windows
    Installing x13binary 
    "C:/PROGRA~1/R/R-32~1.3/bin/x64/R" --no-site-file --no-environ --no-save --no-restore CMD       INSTALL  \
      "C:/Users/tbrooks/AppData/Local/Temp/Rtmpas8erD/devtools33dc6ca24ced/x13org-x13binary-    cf8a9bf" --library="C:/Program Files/R/R-3.2.3/library"  \
      --install-tests 
    
    * installing *source* package 'x13binary' ...
    Warning: running command 'sh ./configure.win' had status 127
    ERROR: configuration failed for package 'x13binary'
    * removing 'C:/Program Files/R/R-3.2.3/library/x13binary'
    * restoring previous 'C:/Program Files/R/R-3.2.3/library/x13binary'
    Error: Command failed (1)

However I installed the latest version of Rtools 3.3, then reboot and retried and it worked.

    devtools::install_github("x13org/x13binary", ref = "bug/quote-windows")
    x13binary::checkX13binary()

Thanks for your help.

**Comment by christophsax** [comment]
Perfect. Which means that the space issue is resolved with the next version of x13binary (on CRAN soon). Thanks everybody.

**Comment by christophsax** [comment]
This should be resolved in the latest CRAN release (0.1.1, available as binary now). To update:

```
install.packages("x13binary")
```
